cmd/terramate/cli/github/types.go

```diff
@@ -17,6 +17,16 @@ type (
 		CreatedAt time.Time `json:"created_at"`
 		UpdatedAt time.Time `json:"updated_at"`
 
+		HEAD PullRequestHEAD `json:"head"`
+
+		// rest of the fields aren't important for the cli.
+	}
+
+	// PullRequestHEAD contains metadata for the PR HEAD.
+	PullRequestHEAD struct {
+		Ref string `json:"ref"`
+		SHA string `json:"sha"`
+
```

Comment:
this was already included in another PR?

Reply:
yes, the PRs are not stacked. I will fix the conflict once the other is merged.